src/pages/country.js

```diff
@@ -0,0 +1,54 @@
+import { useQuery, gql } from "@apollo/client";
+
+export const COUNTRY_QUERY = gql`
+  query getCountry($Id: ID!) {
+    countries(where: { id: { eq: $Id } }) {
+      name
+      vatRate
+      id
+      capital {
+        name
+      }
+      cities {
+        name
+      }
+    }
+  }
+`;
+
+const Country = (props) => {
+
+  const { loading, error, data } = useQuery(COUNTRY_QUERY, {
+    variables: { Id: props.match.params.countryId },
+  });
+
+  if (loading) return <div>Loading...</div>
+  if (error) return <div>something is wrong</div>
+  console.log('dupa') // ;-)
```

Comment:
I like that

Reply:
Glad you liked it, I made it just for you! ;)